gatling-recorder/src/main/scala/io/gatling/recorder/scenario/template/RequestTemplate.scala

```diff
@@ -51,10 +51,12 @@ private[scenario] object RequestTemplate {
         }.getOrElse("")
 
       def renderLongString(value: String) =
-        if (value.nonEmpty)
-          value.grouped(65535).map(protectWithTripleQuotes).mkFastring(" + ")
-        else
-          protectWithTripleQuotes("")
+        if (value.size > MaxLiteralSize) {
+          wrapLongString(value.grouped(MaxLiteralSize).map(protectWithTripleQuotes).mkFastring(", "))
+        } else
+          protectWithTripleQuotes(value)
+
+      def wrapLongString(value: Fastring) = "joinStrings(" + value + ")"
```

Comment:
First, are you sure that just concatenating Strings with + (`"part1" + "part2" + "part3"`) doesn't suffice?
My understanding is just that the compiler parser can't deal with long String literals, as it has a limited buffer.

Then, if that really doesn't work, Scala library has built-in String joining, see `Seq.mkString`.